superset/models/core.py

```diff
@@ -415,22 +425,21 @@ def get_sqla_engine(
                     server_context.local_bind_address,
                 )
                 sqlalchemy_uri = ssh_manager_factory.instance.build_sqla_url(
-                    sqlalchemy_uri, server_context
+                    sqlalchemy_uri,
+                    server_context,
                 )
+
             yield self._get_sqla_engine(
+                catalog=catalog,
                 schema=schema,
                 nullpool=nullpool,
                 source=source,
                 sqlalchemy_uri=sqlalchemy_uri,
             )
 
-    # The `get_sqla_engine_with_context` was renamed to `get_sqla_engine`, but we kept a
-    # reference to the old method to prevent breaking third-party applications.
-    # TODO (betodealmeida): Remove in 5.0
-    get_sqla_engine_with_context = get_sqla_engine
```

Comment:
I removed this because the signature for `get_sqla_engine` changed to accept a catalog.